src/filesystem/fs.rs

```diff
@@ -140,7 +141,7 @@ impl FileSystem {
             return Err(Error::errno_with_msg(
                 Errno::EINVAL,
                 format!(
-                    "current work directory should be a relative path: {:?}",
+                    "current work directory should at least not be a relative path: {:?}",
```

Comment:
> This sounds weird to read. We don't want a relative path anymore?

Yes, passing a relative path to `set_cwd()` can be very odd, especially when initializing proot-rs (when there is no `cwd` yet).

To avoid this problem, we can force that the argument of `set_cwd()` should be an absolute path.

Reply:
If it shouldn't be relative, then we should state that we want an absolute path. The comment should also include this reason for future readers to understand why.

Reply:
> If it shouldn't be relative, then we should state that we want an absolute path. The comment should also include this reason for future readers to understand why.

The reason has been added to the comment.